conf/switches.conf.example

```diff
@@ -6,6 +6,7 @@
 # http://www.fsf.org/licensing/licenses/gpl.html
 
 [default]
+type=Cisco::Catalyst_2960
```

Comment:
Yes but then we have to handle the 'mandatory' case for when there is and there isn't a default type set

It makes the code a bit harder to follow.

What we could do, is allow to use the 'pf::Switch' in the type and make it default so we at least have a base.

For me the default configuration should define all elements of the childs it has and they change them if needed.

I agree choosing the switch is not the best way but using pf::Switch could just be the best way.

Or create a 'Generic Switch' module which we currently lack and assign it as the default type